### DISCUSSION TITLE
Banned in Prime

### DISCUSSION THREAD

**Discussion by PUREHAX** [danielkrupinski/Osiris#2855] [General]
I cheated on a account for a long time in non prime. I was Lvl 13. I used vac bypass everytime and had no issues getting banned. 
Since in non prime it is basically only hvh I bought the prime status and got banned a day after. 

Is it possible that in non prime is no overwatch ? 
An btw I played one match in prime and that was legit. Only chams. How can I get an OW ban ? 
Dont get it

**Comment by gun3kter** [comment]
You can get ban on non prime domo but its unlucky. I have prime 3 months and i still dont have ban.

**Reply by mdng568** [reply]
I have been using Osiris for 1 month now and nothing has happened.

I using Injector - CSGhost v4.3.1 and Osiris

**Comment by linuxer9** [comment]
OW bans are delayed, it could have been an hvh match you played on non-prime.
you can get OW ban in non-prime if you get enough reports.
